hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/ColumnStatsIndexSupport.scala

```diff
@@ -224,7 +228,8 @@ object ColumnStatsIndexSupport {
   private val COLUMN_STATS_INDEX_FILE_COLUMN_NAME = "fileName"
   private val COLUMN_STATS_INDEX_MIN_VALUE_STAT_NAME = "minValue"
   private val COLUMN_STATS_INDEX_MAX_VALUE_STAT_NAME = "maxValue"
-  private val COLUMN_STATS_INDEX_NUM_NULLS_STAT_NAME = "num_nulls"
+  private val COLUMN_STATS_INDEX_NULL_COUNT_STAT_NAME = "nullCount"
```

Comment:
Can we not reuse HoodieMetadataPayload.* constants here as well?

Reply:
Good call. This was copied over from a legacy component w/o much afterthought. Addressed